hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -805,51 +805,79 @@ public boolean isBucketEmpty(String volume, String bucket)
     return deletedKeys;
   }
 
+  /**
+   * @param userName volume owner, null for listing all volumes.
+   * @throws IOException
+   */
   @Override
   public List<OmVolumeArgs> listVolumes(String userName,
       String prefix, String startKey, int maxKeys) throws IOException {
     List<OmVolumeArgs> result = Lists.newArrayList();
```

Comment:
Not introduced in this change, but this list creation can be moved after the `userName` check.

Reply:
Thanks @adoroszlai for the comment.
Fixed.